src/gui/qgsmaptip.cpp

```diff
@@ -200,3 +211,9 @@ QString QgsMapTip::fetchFeature( QgsMapLayer *layer, QgsPoint &mapPosition, QgsM
     return feature.attribute( idx ).toString();
   }
 }
+
+//This slot handles all clicks
+void QgsMapTip::linkClickedSlot( const QUrl &url )
```

Comment:
No, it's just the way we normally call things, it doesn't change anything on the technical side.

I think what you meant was on_mSenderName_signalName with autoconnect. I don't like this way because there's no way an IDE or analyzer could easily introspect the code.